src/net.h

```diff
@@ -213,36 +213,43 @@ class CConnman
     void ForEachNode(Callable&& func)
     {
         LOCK(cs_vNodes);
-        for (auto&& node : vNodes)
-            func(node);
+        for (auto&& node : vNodes) {
```

Comment:
I know it was already there, but can we remove the auto here? It doesnt save any space, and during review I always find myself thinking "OK, gonna go grep for CNode to find every use of it in the codebase", which our ever-increasing use of auto breaks.